core/src/main/java/org/elasticsearch/action/get/GetResponse.java

```diff
@@ -168,4 +169,17 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         getResult.writeTo(out);
     }
+
+	@Override
+	public String toString() {
+		try {
```

Comment:
The second one, implement toString with the utility.
On May 8, 2016 9:28 PM, "Johnny Lim" <notifications@github.com> wrote:

> In core/src/main/java/org/elasticsearch/action/get/GetResponse.java
> <https://github.com/elastic/elasticsearch/pull/18102#discussion_r62442944>
> :
>
> > @@ -168,4 +169,17 @@ public void writeTo(StreamOutput out) throws IOException {
> >          super.writeTo(out);
> >          getResult.writeTo(out);
> >      }
> > +
> > +	@Override
> > +	public String toString() {
> > +		try {
>
> @nik9000 <https://github.com/nik9000> Are you suggesting not implementing
> SearchResponse.toString() but using Strings.toString() in application
> code, or implementing SearchResponse.toString() via using
> Strings.toString()?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/18102/files/c5f0c73b8b0f9c57500656081005aa64e28f509b#r62442944>
>